include/rmm/mr/device/logging_resource_adaptor.hpp

```diff
@@ -283,16 +283,6 @@ class logging_resource_adaptor final : public device_memory_resource {
     return get_upstream_resource() == cast->get_upstream_resource();
   }
 
-  // make_logging_adaptor needs access to private get_default_filename
-  template <typename T>
-  // NOLINTNEXTLINE(readability-redundant-declaration)
-  [[deprecated(
-    "make_logging_adaptor is deprecated in RMM 24.10. Use the logging_resource_adaptor constructor "
-    "instead.")]]
-  friend logging_resource_adaptor<T> make_logging_adaptor(T* upstream,
-                                                          std::string const& filename,
-                                                          bool auto_flush);
-
```

Comment:
Is this right? To my understanding we need the friend declaration (so that `make_logging_adaptor` can access the private method. However, what we can't do is add the `[[deprecated()]]` attribute. So do we want to just delete that one line?

Reply:
If I didn't miss anything, `make_logging_adaptor` calls the public constructor and the public static method `get_default_filename()`, so it does not have to be a friend of `logging_resource_adaptor`. The reason it became a friend is that initially `get_default_filename()` is a private static method.

Reply:
Aha, thanks, I missed that aspet.

Reply:
Thanks for raising that question!